scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -234,14 +234,21 @@ def calc_size(self, sz):
         self.logger.error("Please use valid format Ex: 1024, 1Ki, 1Mi, 1Gi etc..\n")
         return ret
 
-def get_setup_size(self, service):
+def set_setup_size(self, service):
```

Comment:
can we set setup_size as small by default, if it is not any of this case than it would be always small.

Reply:
Resolved in 41f0939a627000ba5a567ba4b037bfb7f013ac2a